app/pages/stacking/components/stacking-info-card.tsx

```diff
@@ -44,7 +48,7 @@ export const StackingInfoCard: FC<StackingInfoCardProps> = props => {
               Cycles
             </Text>
             <ExplainerTooltip>
-              One cycle lasts 1000 blocks on the bitcoin blockchain
+              One cycle lasts 120 blocks on the Bitcoin blockchain
```

Comment:
This isn't a variable provided by the API is it?

Reply:
updated